test/bare.js

```diff
@@ -68,3 +75,72 @@ test('bare inserts __filename,__dirname but not process,global,Buffer', function
         t.equal(code, 0);
     });
 });
+
+test('bare inserts dynamic __filename,__dirname', function (t) {
```

Comment:
Possibly-stupid second question:
The test description mentions both `__filename` and `__dirname`, but AFAICT this is only testing `__dirname`?